pypeit/reduce.py

```diff
@@ -257,7 +260,10 @@ def extract(self, global_sky, sobjs_obj):
                 thismask = self.slitmask == sobj.SLITID  # pixels for this slit
                 inmask = self.sciImg.select_flag(invert=True) & thismask
                 # Do it
-                box_rad = self.par['reduce']['extraction']['boxcar_radius']/plate_scale
+                if sobj.MASKDEF_EXTRACT is True:
+                    box_rad = self.par['reduce']['slitmask']['missing_objs_boxcar_rad'] / plate_scale
```

Comment:
is this `plate_scale` in binned pixels?  just checking.
  I think it should be.

Reply:
Thank for pointing this out @profxj .
It seems to me that this platescale is taken from `self.sciImg.detector.platescale`, which I believe is in the `DetectorContainer` datamodel. The platescale in the `DetectorContainer` is in unbinned pixels. So, it seems I should multiply `plate_scale` for the binning, but that means it should have been done already for `boxcar_radius` and it wasn't. Do you agree?